mllib/src/main/scala/org/apache/spark/mllib/pmml/export/PMMLModelExport.scala

```diff
@@ -44,6 +44,6 @@ private[mllib] trait PMMLModelExport {
     val header = new Header()
       .setApplication(app)
       .setTimestamp(timestamp)
-    new PMML("4.2", header, null)
```

Comment:
In fact, in JPMML 1.4.7 version, the PMML standard was updated to 4.3, so it should be consistent and changed to 4.3 at that time